index.html

```diff
@@ -0,0 +1,352 @@
+            </ul>
+          </li>
+          <li class="list-title">
+            <ul class="lists">
+              <li><h4>Frameworks</h4></li>
+              <li>
+                <img src="assets/icons/expand-icon.png" alt="list expander" />
+              </li>
+            </ul>
+          </li>
+          <li>
+            <img
+              class="list-separator"
+              src="assets/icons/list-separator.png"
+              alt="list-separator"
+            />
+          </li>
+          <li class="list-title">
+            <ul class="lists">
+              <li><h4>Skills</h4></li>
+              <li>
+                <img src="assets/icons/expand-icon.png" alt="list expander" />
+              </li>
+            </ul>
+          </li>
+          <li>
+            <img
+              class="list-separator"
+              src="assets/icons/list-separator.png"
+              alt="list-separator"
+            />
+          </li>
+        </ul>
+      </section>
+      <!---  Footer and Contact-me Form  -->
+      <footer class="footer-container" id="footer-id">
+        <section class="contact-me">
+          <h1 class="contact-me-header">Contact me</h1>
+          <p class="contact-me-message">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I'd love
+            to help with it!
+          </p>
+          <form
+            action="https://formspree.io/f/xzbqoenk"
+            method="post"
+            class="my-contact-form"
+          >
+            <ul class="form-data">
+              <li class="input-username">
+                <input
+                  type="text"
+                  class="form-elements"
+                  name="username"
+                  id="username"
+                  maxlength="30"
+                  placeholder="Peter Parker"
+                  required
+                />
+              </li>
+              <li class="input-email">
+                <input
+                  type="email"
+                  class="form-elements"
+                  name="email"
+                  id="email"
+                  placeholder="Peterparker@gmail.com"
+                  required
+                />
+              </li>
+              <li class="input-comment">
+                <textarea
+                  name="comment"
+                  id="comment"
+                  cols="30"
+                  rows="10"
+                  placeholder="Write your message here"
+                  maxlength="500"
+                  required
+                ></textarea>
+              </li>
+            </ul>
+            <div class="form-btn">
+              <button class="get-in-touch-btn" type="submit">
+                Get In Touch
+              </button>
+            </div>
+          </form>
+          <div class="end-bar">
+            <img src="assets/icons/End-bar.svg" alt="" />
+          </div>
+        </section>
```

Comment:
- [ ] Nice work implementing the "contact-me" section it looks really good :clap: However it would be nice to remove it from the footer. A `<footer>` typically contains information about the author of the section, copyright data or links to related documents.